packages/medusa/src/api/routes/store/carts/create-cart.ts

```diff
@@ -175,6 +176,10 @@ export class StorePostCartReq {
   @Type(() => Item)
   items?: Item[]
 
+  @IsBoolean()
+  @IsOptional()
+  validate_sales_channels = false
```

Comment:
todo: if previous comments validated then we should add the `FeatureFlagDecorators` here

Reply:
thought: This should not be part of the store body as it is not up to the "customer" if the sales channel should be validated or not, correct?